.eslintrc

```diff
@@ -3,7 +3,10 @@
   "parser": "babel-eslint",
   "rules": {
     "no-multiple-empty-lines": 1,
-    "flowtype/require-valid-file-annotation": 2
+    "flowtype/require-valid-file-annotation": 2,
+    "flowtype/boolean-style": 2,
+    "flowtype/space-after-type-colon": 2,
```

Comment:
Good point yeah, Prettier handles this, I'll remove it. Other two are useful, though.